go/pkg/fakes/server.go

```diff
@@ -200,56 +247,62 @@ type OutputFile struct {
 }
 
 // Apply puts the file in the fake CAS and the given ActionResult.
-func (f *OutputFile) Apply(ac *repb.ActionResult, s *Server) {
+func (f *OutputFile) Apply(ac *repb.ActionResult, s *Server, execRoot string) error {
```

Comment:
Do we need to add execRoot here? I'd rather we don't change the Apply() interface just for InputFile. How about putting the full path in InputFile? Or creating a separate function in the TestEnv for creating input files (similar to the execroot package in reclient)?

Reply:
I considered creating something like what we have in re-client. But then this interface is only used in tests, so I decided its ok to change the interface. execRoot is also a common property that I thought it maybe useful for future things (like input directory) that we might add.

Reply:
SGTM